client/app/lib/components/sidebarstacksection/index.coffee

```diff
@@ -106,6 +108,12 @@ module.exports = class SidebarStackSection extends React.Component
       when 'Make Team Default'
         remote.api.JStackTemplate.one { _id: templateId }, (err, template) ->
           computeController.makeTeamDefault template, no  unless err
+      when 'Share Stack With Team'
+        remote.api.JStackTemplate.one { _id: templateId }, (err, template) ->
+          computeController.shareStackWithTeam template  unless err
+      when 'Make Stack Private'
+        remote.api.JStackTemplate.one { _id: templateId }, (err, template) ->
+          computeController.makeStackPrivate template  unless err
```

Comment:
let's move these functions from computeController to JStackTemplate's remote extension and here we can say `template.makePrivate()` and `template.shareStackWithTeam()`